_shared_content/operations_center/integrations/google_cloud.md

```diff
@@ -12,7 +12,7 @@ Follow Google's documentation to [configure a dedicated PubSub receiver](https:/
 
    - Setup a project
    - Create a topic
-   - Add a subscription 
+   - Add a subscription (*you should have the role `logging.admin` explicitly set on your account (for more information, see [associated documentation](https://cloud.google.com/logging/docs/export/configure_export_v2))*)
```

Comment:
To avoid to many parentheses, I suggest separating the first and the `for more information...` with a semi-colon
```suggestion
   - Add a subscription (*you should have the role `logging.admin` explicitly set on your account; for more information, see [associated documentation](https://cloud.google.com/logging/docs/export/configure_export_v2)*)
```